fastai/model.py

```diff
@@ -87,7 +87,7 @@ def set_train_mode(m):
     else: m.train()
 
 
-def fit(model, data, epochs, opt, crit, metrics=None, callbacks=None, stepper=Stepper, **kwargs):
+def fit(model, data, epochs, opt, crit, metrics=None, callbacks=None, stepper=Stepper, all_val=False, **kwargs):
```

Comment:
This all_val thing seems a bit messy and odd - but I don't understand the problem you're solving so perhaps there isn't a better way. Would be interested to hear more.

Reply:
all_val is just to say the fit function we want to validate after each batch.